x-pack/plugins/enterprise_search/public/applications/enterprise_search_content/components/search_index/pipelines/ml_inference/ml_inference_logic.ts

```diff
@@ -375,26 +401,16 @@ export const MLInferenceLogic = kea<
           ...EMPTY_PIPELINE_CONFIGURATION,
         },
         indexName: '',
-        simulateBody: `
-[
-  {
-    "_index": "index",
-    "_id": "id",
-    "_source": {
-      "foo": "bar"
-    }
-  },
-  {
-    "_index": "index",
-    "_id": "id",
-    "_source": {
-      "foo": "baz"
-    }
-  }
+        simulateBody: `[
+
 ]`,
         step: AddInferencePipelineSteps.Configuration,
       },
       {
+        getDocumentApiSuccess: (modal, doc) => ({
+          ...modal,
+          simulateBody: JSON.stringify([doc], undefined, 2),
```

Comment:
did we want to include the entire document or pick just the source field off of it?

Reply:
Simulate endpoint expects the whole document,